client/src/App.vue

```diff
@@ -29,7 +29,7 @@ import SubstraitGraph from "@/components/SubstraitGraph.vue";
         <li class="nav-item">
           <button
             class="nav-link"
-            :class="{ active: $route.path === '/' }"
+            :class="{ active: $route.path === '/' || $route.path.includes('/path/') }"
```

Comment:
this is not working currently. When the user uses the shareable link, the CodeView nav-item is not active. Have to fix this.

Reply:
Fixed this.